src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let sum = 0;
+  const history = [];
```

Comment:
We don't need to store all previous values, we can just create counter, and increase it after each function call